src/Database/Indexes/IndexQuery.ts

```diff
@@ -11,19 +13,16 @@ export class IndexQuery {
   private _defaultOperator?: QueryOperator = null;
   private _waitForNonStaleResults: boolean = false;
   private _waitForNonStaleResultsTimeout?: number = null;
+  private _buffer:string = null;
 
-  constructor(query: string = '', pageSize: number = 128, skippedResults: number = 0,
-    defaultOperator?: QueryOperator, options: IOptionsSet = {}
-  ) {
+  constructor(query: string = '', pageSize: number = 128, skippedResults: number = 0, options: IOptionsSet = {}) {
```

Comment:
PageSize by default now should be int.MaxValue (https://github.com/ravendb/ravendb/blob/v4.0/src/Raven.Client/Documents/Queries/IndexQuery.cs#L183)

 but we should only send it if it was changed by the user:
https://github.com/ravendb/ravendb/blob/v4.0/src/Raven.Client/Documents/Queries/IndexQuery.cs#L211